### PR TITLE
create/complete html skeleton

### PR DESCRIPTION
**Pull request by devtanole**
Screenshot 2024-06-15 at 12 15 13 PM

### CODE REVIEW

File: prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What tags are necessary for a complete HTML Skeleton?
+
+- What type of content belongs within the `<head>` of an HTML document?
+
+- What type of content belongs within the `<body>` of an HTML document?
+
+- Where must the `DOCTYPE` declaration appear in a valid HTML document?
+
```
Comment (shawnkost) [inline prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md]: Don't forget to answer the quiz questions!

**Review by shawnkost — CHANGES_REQUESTED** [review]
Code looks great! Just need to answer the quiz questions

File: prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md (hunk 1 above)
Comment (devtanole) [reply prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md]: Screenshot 2024-06-15 at 12 42 23 PM

**Review by shawnkost — APPROVED** [review]
Approved, but it looks like you accidently included `prep-html-script-tag` in your pull request.

Make sure that if you ever create a branch, you always create that branch while on `main`, or else you create the branch off of another branches state. If you ever need to switch back to a different branch, make sure you `git add` and `git commit` before switching, or else you will pull those files with you

**Pull request merged**